go-demo/scripts/build-version

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+# Generate version.json file following our dockerflow standards
+# "source" : "https://github.com/mozilla-services/Dockerflow",
+# "version": "release tag or string for humans",
+# "commit" : "<git hash>",
+# "build"  : "uri to CI build job"
+# "date"   : "build date of image"
+# Exit immediately if a command exits with a non-zero status
+
+DATE_EPOCH=$(date +%s)
+
+# Detect if running in GitHub Actions
+if [[ "${GITHUB_ACTIONS:-}" == "true" ]]; then
+  SOURCE="${GITHUB_SERVER_URL}/${GITHUB_REPOSITORY}"
+  BUILD_URL="${GITHUB_SERVER_URL}/${GITHUB_REPOSITORY}/actions/runs/${GITHUB_RUN_ID}"
+  REF_NAME="${GITHUB_REF_NAME:-unknown}"
+  COMMIT="${GITHUB_SHA:-unknown}"
+
+  # Handle pull requests explicitly
+  if [[ "${GITHUB_EVENT_NAME:-}" == "pull_request" && -f "${GITHUB_EVENT_PATH:-}" ]]; then
+    COMMIT=$(jq -r '.pull_request.head.sha' < "$GITHUB_EVENT_PATH")
+    REF_NAME=$(jq -r '.pull_request.head.ref' < "$GITHUB_EVENT_PATH")
+  fi
+else
+  # Local defaults
+  SOURCE="local-dev"
```

Comment:
TODO: change this to `mozilla/cicd-demos`

Reply:
[dc202a7](https://github.com/mozilla/cicd-demos/pull/32/commits/dc202a793d45b4198276b840f16414faf19bc1ad)